src/DssProxyActionsCropper.sol

```diff
@@ -368,8 +361,10 @@ contract DssProxyActionsCharter is Common {
         require(CdpRegistryLike(cdpRegistry).owns(cdp) == address(this), "wrong-cdp");
         require(CdpRegistryLike(cdpRegistry).ilks(cdp) == GemJoinLike(gemJoin).ilk(), "wrong-ilk");
 
+        // Exits token amount to proxy address as a token
+        CropperLike(cropper).exit(gemJoin, address(this), amt);
```

Comment:
What is the reasoning for deviating from the other proxy actions here? Is this to keep the rewards in the proxy?

Reply:
yes.

Reply:
Does it make sense to do this? I'm unsure of the intuition on this. Would users expect to collect rewards along the way or do we want to enforce calling `crop()` specifically to pull out all accumulated rewards? It would certainly save gas overall to directly ship the rewards to the EOA all the time.

Reply:
Not sure about the gas savings, see detailed answer below.